paymentsheet-example/src/androidTestDebug/java/com/stripe/android/test/core/ui/PaymentSelection.kt

```diff
@@ -5,17 +5,22 @@ import androidx.compose.ui.test.assertIsDisplayed
 import androidx.compose.ui.test.assertIsEnabled
 import androidx.compose.ui.test.hasText
 import androidx.compose.ui.test.junit4.ComposeTestRule
+import androidx.compose.ui.test.onAllNodesWithTag
 import androidx.compose.ui.test.onNodeWithTag
-import androidx.compose.ui.test.onNodeWithText
 import androidx.compose.ui.test.performClick
-import androidx.compose.ui.test.performScrollTo
 import androidx.compose.ui.test.performScrollToNode
 import androidx.test.platform.app.InstrumentationRegistry
 import com.stripe.android.paymentsheet.TEST_TAG_LIST
 
 class PaymentSelection(val composeTestRule: ComposeTestRule, @StringRes val label: Int) {
     fun click() {
         val resource = InstrumentationRegistry.getInstrumentation().targetContext.resources
+
+        composeTestRule.waitUntil(5000) {
```

Comment:
There's not a "wait until idle" option?

Reply:
There is, but here we are waiting for a particular view to be on scren.